### ISSUE TITLE
Failed creating OpenGL context at version requested. in mupen64plus OS X 10.10

### ISSUE THREAD

**Issue by santiagoitzcoatl** [gonetz/GLideN64#1995]
### I'm using `mupen64plus-video-GLideN64.dylib` which I succesfully compiled from GLideN64-Public_Release_3_0

–

- I got this problem trying to run GLideN64 in OS X 10.10:

**`    Core Error: SDL_SetVideoMode failed: Failed creating OpenGL context at version requested`**

```
Mupen64Plus Console User-Interface Version 2.5.0
UI-Console: attached to core library 'Mupen64Plus Core' version 2.5.0
UI-Console: Includes support for Dynamic Recompiler.
UI-Console: using Video plugin: 'GLideN64' v2.0.0
UI-Console: using Audio plugin: 'Mupen64Plus SDL Audio Plugin' v2.5.0
UI-Console: using Input plugin: 'Mupen64Plus SDL Input Plugin' v2.5.0
UI-Console: using RSP plugin: 'Hacktarux/Azimer High-Level Emulation RSP Plugin' v2.5.0
```

- I'm calling  _run_glide.sh_  like this:

`./mupen64plus.app/Contents/MacOS/mupen64plus --corelib ./mupen64plus.app/Contents/MacOS/libmupen64plus.dylib --plugindir ./mupen64plus.app/Contents/MacOS --gfx mupen64plus-video-GLideN64 "$@"`

–

**`    Is it because I'm in Yosemite?`**

**Comment by gonetz** [comment]
I currently have no access to Mac pc, so I may only guess.
OpenGL context created by emulator, not by the plugin.
GLideN64 requires at least core 3.3 profile.
As I remember, mupen64plus needs SDL 2.0 to set core profile on OS X properly.
You need most recent build of mupen64plus for Mac OS X with SDL 2.0 support.
I don't know where to get it. May be you can build it from sources too.

**Comment by gonetz** [comment]
I've got access to MacBook Pro (Mid 2014) with Mac OS X 10.13.6.
It has Brew installed, so I cloned GLideN64, run cmake and build release version from current master.
Then I downloaded **Mupen64Plus v2.5.9 for Mac** from https://mupen64plus.org/
There are scripts inside mupen64plus-bundle-osx-2 to run the emulator: run_glide.sh and run_rice.sh
I copied my mupen64plus-video-GLideN64.dylib to 
`mupen64plus-bundle-osx-2/mupen64plus.app/Contents/Frameworks/`
Then I copied run_glide.sh to run_gliden64.sh. I opened run_gliden64.sh in text editor and replaced 
`mupen64plus-video-glide64mk2` by `mupen64plus-video-GLideN64`.
I run 
`run_gliden64.sh romname`
from terminal. It runs with no errors. Everything took several minutes.
